packages/xo-web/src/xo-app/index.js

```diff
@@ -154,6 +156,16 @@ export default class XoApp extends Component {
     this.setState({ dismissedSourceBanner: true })
   }
 
+  dismissTrialBanner = () => {
+    cookies.set('dismissedTrialBanner', true, { expires: 1 })
+    this.setState({ dismissedTrialBanner: true })
+  }
+
+  isTrialVersion = createSelector(
+    () => this.props.trial,
+    edition => edition.trial && edition.trial.end > new Date().getTime() && true
```

Comment:
You can use [isTrialRunning](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/xoa-updater.js#L32).